docs/advanced/witness.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 8
+sidebar_label: Witness Node
+title: "Witness Node"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/advanced/witness/"/>
+</head>
+
+_Available as of v1.3.0_
+
+A production Harvester cluster needs control plane to manage nodes and pods. In Harvester, usually with three node management nodes, each node contains the whole control plane components.The etcd is one of the important parts of the control plane. Also, it is used by Kubernetes to store all of its data, including its configuration details, state, and metadata. The etcd nodes have to be an odd number (it's always 3 in Harvester) to maintain the etcd quorum.
+
+Sometimes, a user wants to avoid a management node deploying any workload or user data. Each node runs an etcd service to act as a Kubernetes' etcd cluster member. That's the situation in a Witness node comes into play.
```

Comment:
```suggestion
Harvester clusters deployed in production environments require a control plane for node and pod management. A typical three-node cluster has three management nodes that each contain the complete set of control plane components. One key component is etcd, which Kubernetes uses to store its data (configuration, state, and metadata). The etcd node count must always be an odd number (for example, 3 is the default count in Harvester) to ensure that a quorum is maintained.

Some situations may require you to avoid deploying workloads and user data to management nodes. In these situations, one cluster node can be assigned the *witness* role, which limits it to functioning as an etcd cluster member. The witness node is responsible for establishing a member quorum (a majority of nodes), which must agree on updates to the cluster state.
```

Reply:
These two paragraphs are new so I rewrote them.